docs/community/support.md

```diff
@@ -7,7 +7,7 @@ For more information on our Github repositories and how to contribute, see the [
 
 ## Slack
 We use Slack for discussion and support. For any Screwdriver-related questions, join the `#general` channel on the
-[Screwdriver Slack team](https://screwdriver-cd.slack.com). For everything else, join the `#random` channel. To sign up, use our Slack inviter.
```

Comment:
It might make sense to have a subheading `### Signup` or something

Right now there are 2 different things under 1 header